### PR TITLE
Update indices query with faster call for concise list of all indices

### PR DESCRIPTION
**Pull request by gurbirkalsi**

### CODE REVIEW

File: src/services/datastore.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ export async function queryDatastoreConfig() {
 export async function queryMonthIndices(params) {
   const { datastoreConfig } = params;
 
-  const endpoint = `${datastoreConfig.elasticsearch}/_cat/indices?format=json&pretty=true`;
+  const endpoint = `${datastoreConfig.elasticsearch}/_aliases`;
```
Comment (portante) [inline src/services/datastore.js:32]: Does `_aliases` give you JSON output?

**Review by portante — CHANGES_REQUESTED** [review]

File: src/services/datastore.js (hunk 1 above)
Comment (gurbirkalsi) [reply src/services/datastore.js:32]: Yes, it returns JSON with pretty formatting by default.

Comment (portante) [reply src/services/datastore.js:32]: But our production V1 Elasticsearch instance returns an empty list because we don't use aliases.

Comment (portante) [reply src/services/datastore.js:32]: Okay, I can see how this might be more efficient, as it does return JSON and is a much smaller payload.

**Review by portante — APPROVED** [review]
Looks good.

**Pull request merged**